src/parser/keyword.c

```diff
@@ -526,10 +526,22 @@ static KEYWORD_RECORD keywords[] = {
 static KEYWORD_RECORD *pt_find_keyword (const char *text);
 static int keyword_cmp (const void *k1, const void *k2);
 
+#define GET_KEYWORD_HASH_VALUE(h,s)  do {       \
+  unsigned char* p = (unsigned char*)(s);       \
+  for((h) = 5381;  *p; p++ )                    \
+    {                                           \
+      (h) = (((h) << 5) + (h)) + *p; /* hash * 33 + c */ \
+    }                                           \
+}while(0)
```

Comment:
For code consistency, `do { } while (0)` format as follow.
```suggestion
#define GET_KEYWORD_HASH_VALUE(h,s) \
  do { \
    unsigned char* p = (unsigned char*)(s); \
    for((h) = 5381; *p; p++ ) \
      { \
        (h) = (((h) << 5) + (h)) + *p; /* hash * 33 + c */ \
      } \
} while(0);
```

What is the meaning of 5381?

Reply:
I referenced the existing string_hash().
It is known that there have been research results showing that the distribution efficiency of hash values is good when the algorithm starts with that value when the initial value is 5381.

Reply:
1) we need a reference and description about this value. please add comment
2) please keep the code format for macro function same with others
`}while(0)` -> `} while(0);`
```
#define GET_KEYWORD_HASH_VALUE(h,s)  do {       \

to

#define GET_KEYWORD_HASH_VALUE(h,s) \  do { \
```

Reply:
When defining a macro, it is common not to add';' at the end of the macro for safety reasons.

Reply:
Sorry for typo. I changed the example above. It's about formatting

Reply:
I don't know the exact meaning of the numbers 5381 and 33.
It is a number included in the djb2 algorithm, and I think you can get some related information from the link below.

https://stackoverflow.com/questions/10696223/reason-for-5381-number-in-djb-hash-function/13809282#13809282

Reply:
@hgryoo  OK, I will change indent on macro.